src/components/Transactions/TransactionRow.tsx

```diff
@@ -138,24 +140,18 @@ export const TransactionRow = ({ tx }: { tx: Tx }) => {
               )}
             </Row.Value>
           </Row>
-          <Row variant='vertical'>
-            <Row.Label>
-              <Text translation='transactionRow.network' />
-            </Row.Label>
-            <Row.Value>{tx.network}</Row.Value>
-          </Row>
-          <Row variant='vertical'>
-            <Row.Label>
-              <Text translation='transactionRow.blockHeight' />
-            </Row.Label>
-            <Row.Value>{tx.blockHeight}</Row.Value>
-          </Row>
           <Row variant='vertical'>
             <Row.Label>
               <Text translation={sentTx ? 'transactionRow.to' : 'transactionRow.from'} />
             </Row.Label>
             <Row.Value>
-              <MiddleEllipsis maxWidth='180px'>{tx.to ?? tx.from}</MiddleEllipsis>
+              <Link
+                isExternal
+                color='blue.500'
+                href={`${asset?.explorer}/address/${tx.to ?? tx.from}`}
```

Comment:
i think you can use `asset.explorerTxLink` here

Reply:
@0xdef1cafe actually it wouldn't work use `explorerTxLink` as the _to_ or _from_ are addresses, not transactions.
I didn't find the explorer info for addresses / account in `lib`, only base URL and tx.
Maybe it would require some work at the `lib` / `unchained` level to introduce a new field in asset `explorerAccountLink` or something like that.

Reply:
I just commited a modification to handle Bitcoin chain type for this link.